service-catalog/resources/service-catalog-bucket-cla-b58cf604-019a-467c-8cae-50e3a47c12b1.yaml

```diff
@@ -1,15 +1,16 @@
 apiVersion: backstage.io/v1alpha1
 kind: Resource
 metadata:
-  name: s3-bucket
-  description: AWS S3 bucket deployed using OpenShift
+  name: service-catalog-bucket-cla-b58cf604-019a-467c-8cae-50e3a47c12b1
```

Comment:
Does it make sense to keep the bucketname as the resource / filename? I dont see any links to the resource itself (like the s3 endpoint for instance) so shouldnt we just use a shortend name for it and put the full bucket name in the description if we really need it?

Reply:
Not sure what would you like to see here. Bucket name is the only unique identifier we have for this particular bucket. Keep in mind that this entity describes a specific bucket for Service Catalog. It maps a resource, not a resource type (s3 buckets) in general. 

Yes, there are no links, however there's a relation listed below - this bucket is provided by `odf-external-cluster-smaug` Ceph cluster resource. This cluster is not intended to be accessed from outside of the cluster, so linking `s3.openshift-storage.svc` host doesn't make any sense.